runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/testing/TestDataflowRunnerTest.java

```diff
@@ -95,7 +95,7 @@
   @Mock private MockLowLevelHttpRequest request;
   @Mock private GcsUtil mockGcsUtil;
 
-  private static final String WATERMARK_METRIC_SUFFIX = "windmill-data-watermark";
+  private static final String WATERMARK_METRIC_SUFFIX = "DataWatermark";
```

Comment:
What is this line used for? Should we just make the one in the TestDataflowRunner `@VisibleForTesting`? Also, we should test the legacy case too, right?